### PR TITLE
[IMP] base_wamas_ubl: Support some dict_key on ART telegram

### PR DESCRIPTION
**Pull request by QuocDuong1306**

### CODE REVIEW

File: base_wamas_ubl/lib/wamas/wamas_grammar/art.py [hunk 1]
```diff
@@ -315,15 +315,15 @@
...
+            "dict_key": "sell_only_by_packaging",
             "df_val": False,
             "df_func": False,
         },
         "Art_ChargePflWe": {
             "type": "bool",
             "length": 1,
             "dp": False,
-            "dict_key": False,
+            "dict_key": "wamas_reception_weight",
```
Comment (tuantrantg) [inline base_wamas_ubl/lib/wamas/wamas_grammar/art.py]: @QuocDuong1306 oi, Please remove this one (`collect weight at reception` aka `Art_ChargePflWe` has been crossed off in the document)

File: base_wamas_ubl/lib/wamas/wamas_grammar/art.py [hunk 2]
```diff
@@ -51,7 +51,7 @@
             "type": "str",
             "length": 20,
             "dp": False,
-            "dict_key": False,
+            "dict_key": 'id',
```
Comment (tuantrantg) [inline base_wamas_ubl/lib/wamas/wamas_grammar/art.py]: hi @nilshamerlinck could you please validate this mapping

File: base_wamas_ubl/lib/wamas/wamas_grammar/art.py [hunk 3]
```diff
@@ -67,7 +67,7 @@
             "type": "str",
             "length": 40,
             "dp": False,
-            "dict_key": False,
+            "dict_key": "name",
```
Comment (tuantrantg) [inline base_wamas_ubl/lib/wamas/wamas_grammar/art.py:70]: hi @nilshamerlinck and could you please validate this one also

File: base_wamas_ubl/lib/wamas/wamas_grammar/art.py [hunk 4]
```diff
@@ -315,15 +315,15 @@
             "type": "bool",
             "length": 1,
             "dp": False,
-            "dict_key": False,
+            "dict_key": "sell_only_by_packaging",
```
Comment (tuantrantg) [inline base_wamas_ubl/lib/wamas/wamas_grammar/art.py:318]: hi @nilshamerlinck please validate this mapping too

**Review by tuantrantg — CHANGES_REQUESTED** [review]

File: base_wamas_ubl/lib/wamas/wamas_grammar/art.py (hunk 2 above)
Comment (nilshamerlinck) [reply base_wamas_ubl/lib/wamas/wamas_grammar/art.py]: @QuocDuong1306 is checking example files/integration to confirm mapping

**Comment by tuantrantg** [comment]
This PR's replaced by https://github.com/tuantrantg/edi/pull/15

**Pull request closed without merging**